src/components/views/rooms/MessageComposer.tsx

```diff
@@ -344,6 +347,15 @@ export default class MessageComposer extends React.Component<IProps, IState> {
                         }
                         break;
                     }
+
+                    case "feature_location_share": {
+                        const showLocationButton = SettingsStore.getValue(
+                            "feature_location_share");
```

Comment:
this looks like it would fit on one line

Reply:
Based on "try to keep JavaScript code around the 90 column mark" from the [style doc](https://github.com/matrix-org/matrix-react-sdk/blob/master/code_style.md#general-style) (this is 100 columns), I'd vote for wrapping it.

Reply:
Of course, if I had my way we'd have a hard limit at 80 columns except imports and JSX...